code/modules/food_and_drinks/recipes/recipes_microwave.dm

```diff
@@ -1332,6 +1357,12 @@
 	)
 	result = /obj/item/food/snacks/sashimi
 
+/datum/recipe/microwave/sashimi/make_food(obj/container)
+	var/obj/item/food/snacks/picoss_kabob/being_cooked = ..()
```

Comment:
> Copy-paste error.
> 
> I'm also a bit iffy on this one, because it's _sashimi_. It's supposed to be raw. But it's made in the microwave, so I guess in space, we cook it, and that'd get rid of the toxin? :S

Have you seen how we make ice cream?

Reply:
Fixed